scripts/dartdoc.sh

```diff
@@ -1,5 +1,28 @@
 #!/usr/bin/env bash
 
-cd packages/flame
-flutter pub get
-flutter pub run dartdoc --no-auto-include-dependencies --quiet
+function run_dartdoc() {
+  result=$(flutter pub get 2>&1) # Sadly a pub get can block up our actions as it will retry forever if a package is not found, but this should atleast report everything else.
+  if [ $? -ne 0 ]; then
+    echo "flutter pub get failed:"
+    echo "$result"
+    exit 1
+  fi
```

Comment:
if we really need some sort of wrapper we should do a pub_get.sh script instead of duplicating